project/Dependencies.scala

```diff
@@ -26,7 +26,7 @@ object Dependencies {
     // We don't force Pekko updates because downstream projects can upgrade
     // themselves. For more information see
     // https://pekko.apache.org//docs/pekko/current/project/downstream-upgrade-strategy.html
-    val pekko = "1.0.0-RC3"
+    val pekko = "1.0.0"
     val pekkoBinary = "current"
     val pekkoHttp = "0.0.0+4456-9a4c0fc7-SNAPSHOT"
```

Comment:
when will pekko http be updated to 1.0.0?

Reply:
@pjfanning That's a fair point, should update pekko http to the latest snapshot that depends on pekko 1.0.0

Reply:
rebase is done now